students/Cracolici_Jon/lesson03/JonCracolic_L3_ListLab.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python 3
+#Jon Cracolici
+#Lesson 03 List Lab
+#UW Python Cert
+import copy
+#
+#series 1
+fruitlist1 = ['Apples','Pears','Oranges','Peaches']
+print("Series 1")
+print(fruitlist1)
+fruitlist1.append(input("Please input a new fruit!"))
+print(fruitlist1)
+number = input("Please input a number")
```

Comment:
Add a trailing space, and a colon or similar for readability:  (Please input a number: ") for example.